docs/Enterprise.md

```diff
@@ -0,0 +1,39 @@
+---
+id: enterprise
+title: WebdriverIO for Enterprise
+---
+
+## Available as part of the Tidelift Subscription
+
+Tidelift is working with the maintainers of [Package] and thousands of other
+open source projects to deliver commercial support and maintenance for the open source dependencies you use to build your applications. Save time, reduce risk, and improve code health, while paying the maintainers of the exact dependencies you use.
+
+#### Enterprise-ready open source software—managed for you
+The Tidelift Subscription is a managed open source subscription for application dependencies covering millions of open source projects across JavaScript, Python, Java, PHP, Ruby, .NET, and more.
+
+#### Your subscription includes:
+
+##### Security updates
+Tidelift’s security response team coordinates patches for new breaking security vulnerabilities and alerts immediately through a private channel, so your software supply chain is always secure.
+
+##### Licensing verification and indemnification
+Tidelift verifies license information to enable easy policy enforcement and adds intellectual property indemnification to cover creators and users in case something goes wrong. You always have a 100% up-to-date bill of materials for your dependencies to share with your legal team, customers, or partners.
+
+##### Maintenance and code improvement
+Tidelift ensures the software you rely on keeps working as long as you need it to work. Your managed dependencies are actively maintained and we recruit additional maintainers where required.
+
+##### Package selection and version guidance
+We help you choose the best open source packages from the start—and then guide you through updates to stay on the best releases as new issues arise.
```

Comment:
`start—and` is this correct? it looks a bit strange

Reply:
yeah but it is copied from a template (I doubled checked) and is suppose to be like this.